specutils/io/default_loaders/subaru_pfs_spec.py

```diff
@@ -28,33 +28,52 @@
 
 def spec_identify(origin, *args, **kwargs):
```

Comment:
This may be worth adding to the `/io/default_loaders/parsing_utils.py` file. If `_spec_pattern` is unique between loader definitions, maybe adding in another parameter so that it can passed to the function.

Reply:
To provide a more generic wrapper for `is_fits`, with `_spec_pattern` to be passed by the loader? That might be useful, although I would in general recommend to try to identify the spectrum types on file content rather than naming schemes. But it's useful to have various file type and path checks provided in one function, agreed.

Note that `is_fits` called with `fileobj=None` will identify any valid name pattern as a FITS file, whether `filepath` actually points to an existing file or not. For `_fits_identify_by_name` I have made it a requirement that the file exists, since the loaders will eventually access it anyway.